test/SlowTests/Voron/Storage/StorageReportGenerationTests.cs

```diff
@@ -390,6 +391,54 @@ public void TreeReportContainsInfoAboutStreams(int numberOfStreams, int seed)
             }
         }
 
+        [Theory]
+        [InlineDataWithRandomSeed(1)]
+        [InlineDataWithRandomSeed(5)]
+        [InlineDataWithRandomSeed(15)]
+        public void TreeReportContainsPartialInfoAboutStreams(int numberOfStreams, int seed)
+        {
+            var r = new Random(seed);
+
+            using (var tx = Env.WriteTransaction())
+            {
+                var tree = tx.CreateTree("streams-tree");
+
+                for (int streamNumber = 0; streamNumber < numberOfStreams; streamNumber++)
+                {
+                    var bytes = new byte[r.Next(0, 2 * 1024 * 1024)];
+
+                    r.NextBytes(bytes);
+
+                    var name = $"streams/{streamNumber}";
+                    tree.AddStream(name, new MemoryStream(bytes));
+                }
+
+                tx.Commit();
+            }
+
+            using (var tx = Env.ReadTransaction())
+            {
+                // here we are checking that the original streams' size is ~equal to the calculated [fake] one
+
+                var report = Env.GenerateDetailedReport(tx, includeDetails: false);
+                var fullReport = Env.GenerateDetailedReport(tx, includeDetails: true);
+
+                var treeReport = report.Trees.Find(x => x.Name == "streams-tree");
+                var fullTreeReport = fullReport.Trees.Find(x => x.Name == "streams-tree");
+                
+                Assert.NotNull(treeReport);
+                Assert.NotNull(fullTreeReport);
+
+                Assert.Equal(1, treeReport.Streams.Streams.Count);
+                Assert.Equal(StorageReportGenerator.SkippedStreamsDetailsName, treeReport?.Streams.Streams[0].Name);
+
+                var streamsSizeInMb = Math.Ceiling(treeReport.Streams.Streams[0].AllocatedSpaceInBytes * 0.000001);
```

Comment:
?

Reply:
In `includeDetails: false` case, at the beginning, we are not reading the streams -> we don't have the actual size of each stream. 
For each tree report which contains streams -> we are calculating its size [roughly] by using other parameters we do have - but there may be cases we can miss something on calculation. 
Thus, for skipped streams/trees we can get a difference in sizes [really small one]. For example, we can get 8KB for the calculated ["fake"] value instead of 7.99KB for the original one.